polyfill/lib/ecmascript.mjs

```diff
@@ -492,7 +486,17 @@ export const ES = ObjectAssign({}, ES2022, {
       if (yearString === '-000000') throw new RangeError(`invalid ISO 8601 string: ${isoString}`);
       year = ES.ToIntegerOrInfinity(yearString);
       month = ES.ToIntegerOrInfinity(match[2]);
-      calendar = match[3];
+      const annotations = match[3];
+      for (const [, critical, key, value] of annotations.matchAll(PARSE.annotation)) {
+        if (key === 'u-ca') {
+          if (calendar === undefined) calendar = value;
```

Comment:
I think the `calendar` will always be `undefined` here. Same for `ParseTemporalMonthDayString`.

Reply:
It should normally always be `undefined`, but in the case of a second calendar annotation, it will have a value. The IETF document says that the first one takes precedence.